staging/src/k8s.io/cri-api/README.md

```diff
@@ -0,0 +1,29 @@
+## Purpose
+
+This repository contains type definitions and client code for the Kubernetes
+APIs in the node.k8s.io API group. These APIs are primarily used by the Kubelet
```

Comment:
Do we plan to use this repository to host other node apis?

If yes, we may want to change `cri-api` to something else, e.g. `node-api`;
If no, we should change the README.md

Reply:
And if this repo is only for hosting CRI, we may also want to change the package structure
`k8s.io/cri-api/pkg/apis/cri/runtime/v1alpha2` seems unnecessarily long given that `cri` is the only api to be hosted.

Reply:
Fixed the README.

Here's the break down:
`pkg/apis` is used in the other staging/ repos
`cri` indicates this is the CRI
`runtime` since we may end up adding `streaming` too right under `cri`
`v1alpha2` version number 

As mentioned earlier, let's use this PR to get the infrastructure for the new staging repo ready and we can tweak things until the first milestone :)

Reply:
"streaming" is technically a library, not an API. We can put that in a different place.

How about this? 
`k8s.io/cri-api/pkg/apis/runtime/v1alpha2`

Reply:
Ack. that will drop the extra "cri" +1